lang/cs/Org.Apache.REEF.IMRU.Tests/TestTaskManager.cs

```diff
@@ -623,9 +624,22 @@ private static IActiveContext CreateMockActiveContext(int id)
         /// <returns></returns>
         private static IFailedTask CreateMockFailedTask(string taskId, string errorMsg)
         {
+            return CreateMockFailedTask(taskId, errorMsg, new Exception());
+        }
+
+        /// <summary>
+        /// Creates a mock FailedTask with specified taskId and error message
+        /// </summary>
+        /// <param name="taskId"></param>
+        /// <param name="errorMsg"></param>
+        /// <param name="taskException"></param>
+        /// <returns></returns>
+        private static IFailedTask CreateMockFailedTask(string taskId, string errorMsg, Exception taskException)
```

Comment:
If mock without Exception parameter is to be used when we don't check `AsError`, we could set default value for `taskException` to `null` and use only one version of `CreateMockFailedTask`